doc/api/corepack.md

```diff
@@ -52,35 +52,43 @@ sure that all calls to the relevant binaries are run against the requested
 version, downloading it on demand if needed, and aborting if it cannot be
 successfully retrieved.
 
+You can use [`corepack use`][] to ask Corepack to update your local
+`package.json` to use the package manager of your choice:
+
+```bash
+corepack use pnpm@7.x # sets the latest 7.x version in the package.json
+corepack use yarn@* # sets the latest version in the package.json
+```
+
 ### Upgrading the global versions
 
 When running outside of an existing project (for example when running
 `yarn init`), Corepack will by default use predefined versions roughly
 corresponding to the latest stable releases from each tool. Those versions can
-be overridden by running the [`corepack prepare`][] command along with the
+be overridden by running the [`corepack install`][] command along with the
 package manager version you wish to set:
 
 ```bash
-corepack prepare yarn@x.y.z --activate
+corepack install --global yarn@x.y.z
 ```
 
 Alternately, a tag or range may be used:
 
 ```bash
-corepack prepare pnpm@latest --activate
-corepack prepare yarn@stable --activate
+corepack install --global pnpm@*
+corepack install --global yarn@stable
 ```
 
 ### Offline workflow
 
 Many production environments don't have network access. Since Corepack
 usually downloads the package manager releases straight from their registries,
 it can conflict with such environments. To avoid that happening, call the
-[`corepack prepare`][] command while you still have network access (typically at
+[`corepack install`][] command while you still have network access (typically at
```

Comment:
I can't make a code suggestion on the other lines than the one edited, but this paragraph should be rewritten to mention that `yarn pack` will generate a tarball that can then be installed offline with `corepack install` (and perhaps add a code example).

```suggestion
[`corepack pack`][] command while you still have network access (typically at
```

Reply:
You mean `corepack pack`, not `yarn pack`, right?

Reply:
Sorry, missed the notification - yes indeed